portfolio/src/main/webapp/message-fetch.js

```diff
@@ -0,0 +1,24 @@
+//placeholder mock fetch object until this branch has access to the server
+const fetchMock = () => {
```

Comment:
You should write JS docstrings like https://google.github.io/styleguide/jsguide.html#jsdoc-general-form

Reply:
Removed this function because it's redundant, but added JSDoc on the other function on this file!